src/pages/products/products.jsx

```diff
@@ -30,48 +49,142 @@ return(
+                            type="checkbox" 
+                            name="Balls"
+                            id="balls"
+                            checked = {balls} 
+                            onChange = {() => dispatch({ type:"CATEGORY",categoryValue:"Balls"})} 
+                            />
+                            Balls
+                            </label>
                         
-                        <label><input type="checkbox" name="Stumps" />Stumps</label>
+                        <label>
+                            <input 
+                            type="checkbox" 
+                            name="Pads" 
+                            id="pads"
+                            checked = {pads} 
+                            onChange = {() => dispatch({ type:"CATEGORY",categoryValue:"Pads"})}
+                            />
+                            Pads
+                            </label>
                         
-                        <label><input type="checkbox" name="Gloves" />Gloves</label>
+                        <label>
+                            <input 
+                            type="checkbox" 
+                            name="Gloves" 
+                            id="gloves"
+                            checked = {gloves} 
+                            onChange = {() => dispatch({ type:"CATEGORY",categoryValue:"Gloves"})}
+                            />
+                            Gloves
+                            </label>
                     </div>
                     
                     <h3 className="margin-top">Rating</h3>
-                    <div className="flex-column">
+                    <div className="flex-column element-spacing">
                         
-                    <label htmlFor="rating-4"><input type="radio" name="rating" value="4" id="rating-4" />4 star and above</label>
+                    <label htmlFor="rating-4">
+                        <input 
+                        type="radio"
+                        name="rating"
+                        value="4" 
+                        id="rating-4"
+                        checked = {rating === 4}
+                        onChange = {() => dispatch({ type:"RATING", ratingValue:"4"})} 
+                        />
+                        4 star and above
+                        </label>
                     
-                    <label htmlFor="rating-3"><input type="radio" name="rating" value="3" id="rating-3" />3 star and above</label>
+                    <label htmlFor="rating-3">
+                        <input 
+                        type="radio" 
+                        name="rating" 
+                        value="3" 
+                        id="rating-3"
+                        checked = {rating === 3}
+                        onChange = {() => dispatch({ type:"RATING", ratingValue:"3"})} 
+                        />
+                        3 star and above
+                        </label>
```

Comment:
For rating, you can use the map function. It will reduce your lines of code too

Reply:
okay